app/client/widgets/ChoiceTextBox.ts

```diff
@@ -85,6 +90,7 @@ export class ChoiceTextBox extends NTextBox {
       cssRow(
         dom.autoDispose(disabled),
         dom.autoDispose(mixed),
+        cssChoiceEditIcon('Dropdown'),
```

Comment:
```suggestion
```

The config DOM is what's shown in the right panel (with the editor for choice options), so we don't want to add the icon here.

Reply:
Weird, I thought that I have pushed the change… I fix that!

Reply:
I hope that's better now, thank you for your vigilance!